translations/ee.json

```diff
@@ -0,0 +1,31 @@
+{
+    "navbar_home": "Avaleht",
+    "navbar_services": "Teenused",
+    "navbar_about": "Meist",
+    "navbar_contact": "Kontakt",
+
+    "cctv_title": "Meie Kunst - Luua Turvalisi Nutikodusid",
+    "cctv_subtitle": "Valige usaldusväärsed ja nutikad lahendused ettevõttelt U SWITCH, et teha oma kodust veelgi mugavam ja turvalisem paik.",
+
+    "services_header": "Pakume innovaatilisi valve- ja targa kodu lahendusi kasutades <a class='underline' href='https://ajax.systems/' target='_blank' title='https://ajax.systems/'>AJAX tooteid</a>.",
+
+    "services_service_paragraph": "TEENUS",
+    "services_smart_module_setup": "Nutimoodulite paigaldus",
+    "services_smart_module_setup_description": "Paigaldame nutimooduleid 12 tunni jooksul",
+
+    "services_smarthouse_solutions": "Nutimaja lahendused",
+    "services_smarthouse_solutions_description": "Teostame töid alati lähtudes kliendi soovidest ja tagasisidest",
+
+    "devices_sell": "Seadmete müük",
+    "devices_sell_description": "Pakume tipptasemel juhtmevabu tooteid, mille on innovaatiline disain",
+
+    "about_title": "Meist",
+    "about_paragraph_1": "Ettevõte U Switch on spetsialiseerunud nutimajade ehitamisele, nutimoodulite paigaldamisele ning kvaliteetsete seadmete müügile. Oleme Ajax Security System kööstööpartnerid, tänu millele saame pidevalt nende poolt täiendavaid koolitusi, tagades sellega oma kliendile kõige uuema ja turvalisema lahenduse.",
+    "about_paragraph_2": "Meie eesmärk on muuta Teie kodu või äripind turvalisemaks ning lihtsustada igapäevaseid toiminguid nutikate seadmete abil. Oleme pühendunud pakkuma parimat teenust ning kvaliteetseid tooteid. Liituge meiega ja laske meil teha Teie elu lihtsamaks ja turvalisemaks!",
+    "about_paragraph_3": "                      - Denis Kitšigin ja Pavel Borovik (Juhatuse liikmed)",
+
+    "contact_title": "Kontakt",
+    "contact_register_number": "Registrikood: 16795378",
+
+    "book_time_button":"Broneeri konsultatsioon"
+}
```

Comment:
I checked that the JSONs also could be formated better:
<img width="790" alt="image" src="https://github.com/user-attachments/assets/5718d85e-d910-4497-8724-f704060d1726">